Source/WebCore/page/cocoa/MemoryReleaseCocoa.mm

```diff
@@ -47,8 +47,6 @@
 void platformReleaseMemory(Critical)
 {
 #if PLATFORM(IOS_FAMILY) && !PLATFORM(IOS_FAMILY_SIMULATOR) && !PLATFORM(MACCATALYST)
-    // FIXME: Remove this call to GSFontInitialize() once <rdar://problem/32886715> is fixed.
-    GSFontInitialize();
```

Comment:
I'm afraid that <rdar://problem/32886715> is not resolved, and the underlying reason @cdumez added this call is still relevant. We should not remove this code.